### ISSUE TITLE
Support add torrent with web-seeding uri from command line option

### ISSUE THREAD

**Issue by multiSnow** [aria2/aria2#1011]
It is easy to add torrent with RPC interface, which is documented in the manpage.
But web-seeding is not mentioned in 'BitTorrent/Metalink Options', only see a line:
```
Downloading a multi-file torrent while also using HTTP/FTP is not supported.
```
which shows that something like below would not work:
```
$ aria2 files.torrent https://home/files/
```

So, is it possible to do a BitTorrent download with web-seeding from command line?

**Comment by tatsuhiro-t** [comment]
If you can do it with RPC interface, it should be possible with command-line.
Have you tried that?  Perhaps, the documentation is not updated.

**Comment by multiSnow** [comment]
Something like below download the torrent file and download the page from http link, but not do like web-seeding, tested:
```
$ aria2 files.torrent https://home/files/
```

**Comment by tatsuhiro-t** [comment]
Try this:

```
$ aria2 -M files.torrent https://home/files/
```

**Comment by multiSnow** [comment]
Yes, it works. Thanks

**Comment by tatsuhiro-t** [comment]
Documentation has been fixed by the commit 31a2f5cd1b1293c0d08efb38a0b9cfff0df38976
